src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
```

Comment:
We don't need this empty lines here. Try to avoid adding redundant empty lines to your code.

Reply:
You can create your Ball in one line using second constructor:
Ball ball = new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));